command/secrets/init/secrets_init.go

```diff
@@ -1,25 +1,71 @@
 package init
 
 import (
+	"fmt"
+
 	"github.com/0xPolygon/polygon-edge/command"
 	"github.com/spf13/cobra"
 )
 
+const (
+	// maxInitNum is the maximum value for "num" flag
+	maxInitNum = 1000
```

Comment:
Personally, I would make maxInitNum a much lower value, like 30. But it's optional, so if you don't think we should change this just resolve comment :D

Reply:
Fixed: https://github.com/0xPolygon/polygon-edge/pull/770/commits/df3173e8ff57e8e2ef4bc4f66b8b4f077087a325